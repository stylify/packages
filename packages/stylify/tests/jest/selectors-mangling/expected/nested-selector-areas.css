:root {--lighten-fff-20: #ffffff;}.d{background-color:var(--lighten-fff-20)}.a{color:blue}.c{color:blue1}.e{color:orange}.b{font-weight:bold}
