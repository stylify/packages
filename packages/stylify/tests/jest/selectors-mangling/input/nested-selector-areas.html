<!-- Astro syntax -->
<div class={font-weight:bold color:blue1 background-color:lighten(#fff,20)}>
	<span class="color:blue"></span>
	<div class={color:orange}></div>
</div>
