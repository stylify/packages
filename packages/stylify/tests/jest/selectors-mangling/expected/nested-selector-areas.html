<!-- Astro syntax -->
<div class={b c d}>
	<span class="a"></span>
	<div class={e}></div>
</div>
